packages/tools/accessibility/README.md

```diff
@@ -0,0 +1,88 @@
+
+The **Accessibility Package** provides a way to create the accessibility tree, which consists of HTML Twins for objects in the scene that should be accessible. Here's an example of a simple scene of three boxes using the Accessibility Package. If you turn on a screen reader to read the page, it will say "A big box in the middle of the scene. A small box on the left of the big box. A small box on the right of the big box".
+
+Example:
+https://playground.babylonjs.com/?snapshot=refs/pull/12074/merge#C9GZTF#1
+
+## How to use Accessibility Package to Support Screen Reader and Keyboard Navigation
+
+### IAccessibilityTag
+
+Blind or low vision users listen to the content and use keyboard to interact, so we need to add description to BabylonJS content that will be read by screen reader. You can add description to your Nodes or Controls (that you think should be accessible) by using **IAccessibilityTag**.
+
+```javascript
+let egg = BABYLON.MeshBuilder.CreateSphere("Egg", {diameterX: 0.62, diameterY: 0.8, diameterZ: 0.6}, scene);
+egg.accessibilityTag = {
+    description: "An easter egg"
+}
+```
+
+Not all content in the scene should be accessible. For example, the decorative trees or background image on a UI panel. Only add IAccessibilityTag to the contents that's important to the user experience.
+
+By default, all Controls (GUI) are considered "important" for accessibility, and will be rendered in accessibility tree with its own information (like text to show on a button), even you hasn't assigned the IAccessibility tag. But if you define the IAccessibilityTag, it can override default metadata (like assigning the IAccessibilityTag.description will override the text to show on a button). By default, Node type object are not considered "important", unless you assign a IAccessibilityTag to it.
+
+### RenderAccessibilityTree()
+
+The accessibility package basically generates HTML elements for your scene, based on the metadata (IAccessibilityTag) you added to your scene content. The screen reader then can read the HTML elements. To generate it for your scene:
+
+```javascript
+ACCESSIBILITY.AccessibilityRenderer.RenderAccessibilityTree(scene);
+```
+
+This will generate a `<div id="accessibility-tree">` HTML element right after the babylonJS scene's canvas element. Inside this div element, the renderer generate an HTML twin for each of your accessible contents in the scene. These HTML twin elements are internally connected with BabylonJS objects, so the screen reader user can also interact with the BabylonJS objects through their HTML twin.

```

Comment:
"...the renderer generate an HTML twin..." the verb should be for a singular subject:

"...the renderer **generates** an HTML twin..."